build.properties

```diff
@@ -94,7 +94,7 @@ phpunit.sparql_url = sparql://${sparql.host}:${sparql.port}/
 # -------------------------
 
 # Apache Solr version to use.
-solr.version = 6.2.1
+solr.version = 6.3.0
```

Comment:
Cool that we updated, but why is the previous broken? is it because it is removed after a newer version is being released?